src/uct/ib/rc/accel/rc_mlx5.inl

```diff
@@ -1687,7 +1694,8 @@ static ucs_status_t UCS_F_ALWAYS_INLINE uct_rc_mlx5_common_ep_short_iov_dm(
                                        hdr_len + iov_length, rdma_raddr,
                                        rdma_rkey, av_size, fm_ce_se, 0, desc,
                                        buffer,
-                                       (log_sge.num_sge > 0) ? &log_sge : NULL);
+                                       (log_sge.num_sge > 0) ? &log_sge : NULL,
+                                       dci_channel);
```

Comment:
IMO better add dci_channel after fm_ce_se in all functions since it's also part of the control seg

Reply:
> IMO better add dci_channel after fm_ce_se in all functions since it's also part of the control seg

ok, but it make changes **much** more complicated for verification  :)